spec/models/blog_post_spec.rb

```diff
@@ -0,0 +1,68 @@
+RSpec.describe BlogPost do
+  describe "#new_content_for_agent?" do
+    subject(:new_content_for_agent?) { described_class.new_content_for_agent?(agent) }
+
+    context "when no post in db" do
+      context "when agent has never read the blog posts" do
+        let(:agent) { Agent.new(blog_read_at: nil) }
+
+        it { is_expected.to be(false) }
+      end
+
+      context "when agent has read the news before (somehow)" do
+        let(:agent) { Agent.new(blog_read_at: 2.minutes.ago) }
+
+        it { is_expected.to be(false) }
+      end
+    end
+
+    context "when their is a post in db" do
```

Comment:
hello this is captain nitpick

```suggestion
    context "when there is a post in db" do
```